### DISCUSSION TITLE
Next newsletter edition: Keyboard shortcuts ✂️

### DISCUSSION THREAD

**Discussion by tuves** [community/community#72086] [Announcements]
When life is easy for developers, good things happen. 😍

Which is why the next Insider newsletter will explore keyboard shortcuts. 👏

Explore a bundle of useful shortcuts you can use on GitHub—so you can work quickly and focus on stuff that matters. 🔥

**[Don't miss it! Subscribe now](https://resources.github.com/newsletter/)**

![image](https://github.com/community/community/assets/49915749/526d9e39-51f6-4d0b-b74d-ee70fe945c68)

**Comment by kachuyjmve** [comment]
Yiii I'm tired and hungry Facebook can we continue with when they smarter played a week cuz I got a lot of stuff to do thank you

**Comment by kachuyjmve** [comment]
I know I know where there is a lot of stuff to finish in the software and the trail for the Google console in the club and set up the attribute for the account and have a split them I'm working on the cell phone so it's very hard to type and write commands and get the the paperwork so give me a break and I'll probably back soon

**Comment by Thebeast44** [comment]
Thanks frendsss

**Comment by parasshinde** [comment]
Thanks for this newsletter!!

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
